nextflow.config

```diff
@@ -11,34 +11,37 @@ params { includeConfig 'default_params.config' }
 
 process {
 
+    //Default values if a label hasn't been specified within a process
     cpus = 4
     memory = 4.GB
+    errorStrategy = { task.attempt < 3 ? 'retry' : 'ignore' }
```

Comment:
I've opted for the errorStrategy to `ignore` that particular sample (process) if the second attempt fails. This way we could reduce the failure of overall pipeline in case of a sample failure.

This strategy was already available in `pbs` and `server` executors, I've made it the default now.